pkg/kubelet/container/helpers.go

```diff
@@ -396,6 +396,8 @@ func MakePortMappings(container *v1.Container) (ports []PortMapping) {
 
 // HasAnyRegularContainerStarted returns true if any regular container has
 // started, which indicates all init containers have been initialized.
+// FIXME: This function is not accurate when its pod sandbox is recreated. Use
```

Comment:
nit: I think this function won't be fixed as stated in the comment, so using `Deprecated:` might be more appropriate than `FIXME:`.

Reply:
fixed.